src/cli/help_menu.js

```diff
@@ -0,0 +1,27 @@
+export function getHelpMenu() {
+    console.log("\n\n", "Options:", "\n\n\n",
```

Comment:
you should use a https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals here for the string as it's easy to write and read.

Reply:
yeah, it's way easier thanks! :D